runcellpose.py

```diff
@@ -200,10 +229,37 @@ def set_directory_fn(path):
             doc="""\
 Fraction of the GPU memory share available to each worker. Value should be set such that this number times the number
 of workers in each copy of CellProfiler times the number of copies of CellProfiler running (if applicable) is <1
+""",
+        )
+        
+        self.stitch_threshold = Float(

```

Comment:
Does this setting do anything if do_3D is true? If not, you could consider making it not visible when do_3D is set.